src/Todoist.Net/Models/BaseEntity.cs

```diff
@@ -22,16 +23,8 @@ internal BaseEntity()
         /// <value>
         /// The identifier.
         /// </value>
-        [JsonProperty("id")]
+        [JsonPropertyName("id")]
+        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
```

Comment:
I suppose this is a breaking change? The default value for the object is null. But the previous behavior checked if the object is empty.

Reply:
I double-checked and found that the default value for the `ComplexId` struct is in fact empty:

![image](https://github.com/olsh/todoist-net/assets/44468352/5e12daa7-4484-4464-94e7-93e7231763fb)